benchmark/README.md

```diff
@@ -42,6 +42,7 @@ TPCH timing results is written to stdout in the following form: `TPCH_Result,<la
 
 ## CSharp
 1. Ensure that the Microsoft.Spark.Worker is properly [installed](../deployment/README.md#cloud-deployment) in your cluster.
+2. Ensure that you build the worker and application with .NET Core 3.0 in order to run hardware acceleration queries.

```

Comment:
I would update this PR in anticipation for netcoreapp3.1 and just update `Tpch.csproj` to reflect that.  

This bullet, can just be a Note at the bottom of the `CSharp` section.

Reply:
Done